fdbclient/GetEncryptCipherKeys.cpp

```diff
@@ -0,0 +1,23 @@
+/*
+ * GetEncryptCipherKeys.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2022 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "fdbclient/GetEncryptCipherKeys.actor.h"
+
+BlobCipherKeyCacheMetrics* g_blobCipherKeyCacheMetrics = nullptr;
```

Comment:
seems like we need this .cpp just for declaring a single variable, wish there is a better to way to get it done..